lib/types/index.js

```diff
@@ -481,7 +481,7 @@ FrameHeader.getProtocolVersion = function (buffer) {
  * @returns {FrameHeader}
  */
 FrameHeader.fromBuffer = function (buf, offset) {
-    let streamId = 0;
+    let streamId;
```

Comment:
Is it even used?
If no, why doesn't it trigger eslint?

Reply:
Yeah, it is used: 
![image](https://github.com/user-attachments/assets/fae8639b-42b1-45b9-aa59-afd1b2b2c7f5)

Reply:
I'm totally blind